Fix unrecognized option '--all'

To reproduce use: 

 service networking restart

This calls:

  ifup -a --exclude=lo ("service networking restart")

which sets $ADDRFAM == "meta" and $IFACE == "--all"


Index: new/etc/network/if-up.d/vzifup-post.in
===================================================================
--- new.orig/etc/network/if-up.d/vzifup-post.in	2012-09-28 09:05:45.000000000 +0200
+++ new/etc/network/if-up.d/vzifup-post.in	2014-03-13 07:38:18.415372011 +0100
@@ -1,3 +1,5 @@
 #!/bin/sh
 
+[ "$ADDRFAM" = "inet" ] || [ "$ADDRFAM" = "inet6" ] || exit 0
+
 exec @SBINDIR@/vzifup-post "$IFACE"
